instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/ui/routeline/SetRouteOrderTest.kt

```diff
@@ -62,35 +62,29 @@ class SetRouteOrderTest : BaseTest<BasicNavigationViewActivity>(
         val shortRoutes = listOf(RouteLine(shortRoute, null))
 
         val consumerLongRoute =
-            object : MapboxNavigationConsumer<Expected<RouteSetValue, RouteLineError>> {
-                override fun accept(value: Expected<RouteSetValue, RouteLineError>) {
-                    LoggerProvider.logger.e(Tag("SetRouteCancellationTest"), Message("long"))
-                    val primaryRoute = routeLineApi.getPrimaryRoute()
-                    val contents = (value as Expected.Success).value
-                        .trafficLineExpression.contents as ArrayList<*>
-                    assertEquals(
-                        625,
-                        contents.size
-                    )
-                    assertEquals(longRoute, primaryRoute)
-                    myResourceIdler.decrement()
-                }
+            MapboxNavigationConsumer<Expected<RouteLineError, RouteSetValue>> { value ->
+                LoggerProvider.logger.e(Tag("SetRouteCancellationTest"), Message("long"))
```

Comment:
Curious: Why are we logging in a test? 🤔 

cc @cafesilencio

Reply:
I guess it could become useful if the test ever becomes flaky.